test/unit/models/test_oauth2.py

```diff
@@ -21,10 +21,10 @@ class TestCreateAccessToken(Test):
     @patch("server.models.oauth2.env")
     @patch("server.models.oauth2.encode")
     def test_should_encode_access_token(self, encode, env):
-        secret_key = self.faker.sha256()
+        SECRET_KEY = self.faker.sha256()
```

Comment:
Aqui pode manter o nome da variável em caixa baixa.

```suggestion
        secret_key = self.faker.sha256()
```